lib/ex_doc/utils.ex

```diff
@@ -8,6 +8,18 @@ defmodule ExDoc.Utils do
     apply(m, f, [module | a])
   end
 
+  def before_closing_head_tag(%{before_closing_head_tag: before_closing_head_tag}, module)
+      when is_map(before_closing_head_tag) do
+    case Map.get(before_closing_head_tag, module) do
+      nil ->
+        raise ArgumentError,
+              "#{inspect(module)} is not defined in before_closing_head_tag configuration"
+
+      value ->
+        value
+    end
```

Comment:
I think for convenience we can say the key defaults to an empty string:

```suggestion
    Map.get(before_closing_head_tag, module, "")
```